Dockerfile

```diff
@@ -329,6 +343,7 @@ RUN find /out -name "*.a" -delete -or -name "*.la" -delete
 FROM alpine:${ALPINE_VERSION}
 LABEL maintainer="Roman Volosatovs <rvolosatovs@riseup.net>"
 COPY --from=upx /out/ /
+COPY --from=protoc_gen_ts /out/ /
```

Comment:
Why not compress with UPX like other plugins?

Reply:
Discussed with markin-io and agreed to compress with UPX, tested working locally